.gitignore

```diff
@@ -360,4 +360,5 @@ build/
 dist/
```

Comment:
@MVliegenthart Please remove the files which are not related to your connector.

Reply:
@sriyen-msft I think I've managed to remove the .gitignore file (sorry I'm quite new to Github).